features/autoscope.feature

```diff
@@ -0,0 +1,258 @@
+Feature: Autoscope
+
+Background:
+   Given I'm in project dir
+
+Scenario: Render uses the scope of the imported Oyafile by default
+  Given file ./Oyafile containing
+    """
+    Project: project
+
+    Require:
+      github.com/test/foo: v0.0.1
+
+    Import:
+      foo: github.com/test/foo
+
+    Values:
+      fruit: apple
+    """
+  And file ./.oya/packs/github.com/test/foo@v0.0.1/Oyafile containing
+    """
+    Values:
+      fruit: orange
+
+    render: |
+      set -e
+      oya render ./templates/file.txt
+    """
+  And file ./templates/file.txt containing
+    """
+    <%= fruit %>
+    """
+  When I run "oya run foo.render"
+  Then the command succeeds
+  And file ./file.txt contains
+  """
+  orange
```

Comment:
Is it really the semantics we want? This makes it impossible to override the values used in a child task, no?